BLSQ/iaso
# WC2-167 search feature on entity api

Mobile specific API to serve entity types /api/mobile/entitytypes

You can retrieve entities from entity types by /api/mobile/entitypes/{entity_type_id}/entities
filter on date by ?limit_date
json logic by ?json_content

Related JIRA tickets : WC2-167

## Self proofreading checklist



## How to test

Create entity types, entities from different accoun, play with the api and check the data
